promql/testdata/aggregators.test

```diff
@@ -497,3 +497,13 @@ eval instant at 1m avg(data{test="-big"})
 
 eval instant at 1m avg(data{test="bigzero"})
 	{} 0
+
+clear
+
+# Test that aggregations are deterministic.
+load 10s
+	up{job="prometheus"} 1
+	up{job="prometheus2"} 1
+
+eval instant at 1m count(topk(1,max(up) without()) ==  topk(1,max(up) without()))
+	{} 1
```

Comment:
Also a lot more concise than my example.

Wondering if there's a way to make it a lot more likely for the test to catch non-determinism though. Best I can come up with is copy-pasting the `eval` statement a few times.

Reply:
It did fail 100% of the time on my machine.

Reply:
Note that every eval is evaluated multiple times: with an instant query, with a query range, with a @ timestamp.